python/ray/_private/utils.py

```diff
@@ -1304,7 +1308,7 @@ def get_release_wheel_url(
     ray_commit: str = ray.__commit__,
     sys_platform: str = sys.platform,
     ray_version: str = ray.__version__,
-    py_version: str = f"{sys.version_info.major}{sys.version_info.minor}",
+    py_version: Tuple[int, int] = sys.version_info[:2],
```

Comment:
why do we fix this? The previous code seems to be easier to understand

Reply:
No strong reason, I just think it's better not to convert it to a string until the very end of the data flow, when we actually generate the wheel name.

But I agree that `sys.version_info[:2]` is harder to understand. I'll change it to `(sys.version_info.major, sys.version_info.minor)`